doc/rados/configuration/filesystem-recommendations.rst

```diff
@@ -29,28 +29,46 @@ Filesystems
 Ceph OSD Daemons rely heavily upon the stability and performance of the
 underlying filesystem.
 
-.. note:: We currently recommend ``XFS`` for production deployments.
-   We recommend ``btrfs`` for testing, development, and any
-   non-critical deployments.  We believe that ``btrfs`` has the correct
-   feature set and roadmap to serve Ceph in the long-term, but ``XFS``
-   and ``ext4`` provide the necessary stability for today's deployments.
-   ``btrfs`` development is proceeding rapidly: users should be
-   comfortable installing the latest released upstream kernels and be
-   able to track development activity for critical bug fixes.
-
-Ceph OSD Daemons depend on the Extended Attributes (XATTRs) of the underlying
-file system for various forms of internal object state and metadata. The
-underlying filesystem must provide sufficient capacity for XATTRs.  ``btrfs``
-does not bound the total xattr metadata stored with a file.  ``XFS`` has a
-relatively large limit (64 KB) that most deployments won't encounter, but the
-``ext4`` is too small to be usable.
+Recommended
+-----------
+
+We currently recommend ``XFS`` for production deployments.
+
+We used to recommend ``btrfs`` for testing, development, and any non-critical
+deployments becuase it has the most promising set of features.  However, we
+now plan to avoid using a kernel file system entirely with the new BlueStore
+backend.  ``btrfs`` is still supported and has a comparatively compelling
+set of features, but be mindful of its stability and support status in your
+Linux distribution.
+
+Not recommended
+---------------
+
+We recommend *against* using ``ext4`` due to limitations in the size
+of xattrs it can store, and the problems this causes with the way Ceph
+handles long RADOS object names.  Although these issues will generally
+not surface with Ceph clusters using only short object names (e.g., an
+RBD workload that does not include long RBD image names), other users
+like RGW make extensive use of long object names and can break.
+
+Starting with the Jewel release, the ``ceph-osd`` will refuse to start
```

Comment:
Reads like it is missing "daemon" after `ceph-osd`
   the `ceph-osd` deamon

Or without it:

    ... the Jewel release, ``ceph-osd`` will refuse to start